src/utils/Shell.ts

```diff
@@ -105,12 +105,39 @@ class ZSH extends Shell {
     }
 }
 
+class Cmd extends Shell {
+    get executableName() {
+        return "cmd.exe";
+    }
+
+    get configFiles() {
+        return [
+        ];
+    }
+
+    get noConfigSwitches() {
+        return [];
+    }
+
+    get preCommandModifiers(): string[] {
+        return [];
+    }
+
+    get historyFileName(): string {
+        return "";
+    }
+}
+
 const supportedShells: Dictionary<Shell> = {
     bash: new Bash(),
     zsh: new ZSH() ,
+    'cmd.exe': new Cmd()
 };
 
 const shell = () => {
+    if (!process.env.SHELL) {
+        process.env.SHELL = 'C:\\Windows\\System32\\cmd.exe';
+    }
```

Comment:
`$SHELL` typically isn't set on Windows, they have a similar environment variable `COMSPEC` but it defaults to the 32-bit version of cmd.exe, as does "cmd.exe" on the path. I've found defaulting to `%windir%\Sysnative\cmd.exe` on 64-bit machine and `%windir%\System32\cmd.exe` on 32-bit machines is the best way to go